ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md

```diff
@@ -0,0 +1,330 @@
+Additionally, there have
+been no changes to the organization criteria as submitted in the PAS submitter application TBD: ApplicationID.
+
+### DOCUMENT RELATED CRITERIA (SD9 7.4)
+
+The DFDL Specification provides a
+modeling language for describing general text and binary data in a standard way. A DFDL model or 
+schema allows any text or binary data to be read (or "parsed") from its native format and to
+be presented as an instance of an information set. (An information set is a logical 
+representation of the data contents, independent of the physical format. 
+For example, two records could be in different formats, because one has fixed-length fields 
+and the other uses delimiters, but they could contain exactly the same data, and would both be 
+represented by the same information set). 
+The same DFDL schema also allows data to be taken from an instance of an information set and 
+written out (or "serialized") to its native format.
+
+This specification addresses a specific challenge faced response to a need for interoperable APIs 
+to be able to understand data regardless of source. 
+A language was needed capable of modeling a wide variety of existing text and binary data formats. 
+A working group was established at the Global Grid Forum (which later became the Open Grid Forum) 
+in 2003 to create a specification for such a language which has now become a Full Recommendation of the 
+OGF. 
+
+This submission addresses the document related criteria specified in SD 9 clause 7.4 as follows:
+
+#### 7.4.1 Quality
+_Within its scope the specification shall completely describe the functionality (in terms of interfaces,
+protocols, formats, etc.) necessary for an implementation of the submission. 
+If it is based on a product, it
+shall include all the functionality necessary to achieve the stated level of compatibility or interoperability
+in a product independent manner._
+
+##### 7.4.1.1 Completeness (M)
+_a) How well are all interfaces specified?_
+
+The DFDL Specification has multiple implementations including several different implementations by IBM 
+corporation, DFDL4S by the European Space Agency, and an open-source implementation (Apache Daffodil). 
+Interoperability of the IBM Java-based DFDL implementation and the Apache Daffodil implementation has been 
+demonstrated for several DFDL format descriptions (called 'schemas'). 
+
+_b) How easily can implementation take place without need of additional descriptions?_
+
+The specification is self-contained. 
+It references other documents which collectively provide everything required to implement the language. 
+
+_c) What proof exists for successful implementations (e.g. availability of test results for media standards)?_
+
+IBM has two independent DFDL implementations. One is for a product called App Connect Enterprise (ACE), the other for the z/TPF system.
+Both have hundreds of customer deployments. 
```

Comment:
z/TPF only has about 30 (very large!) customers

Reply:
I will change the wording to reflect combined number of hundreds of customers.